src/server/svr_func.c

```diff
@@ -7180,18 +7191,24 @@ action_opt_bf_fuzzy(attribute *pattr, void *pobj, int actmode)
 {
 	char *str = pattr->at_val.at_str;
 
-	if (str == NULL)
-		return PBSE_BADATVAL;
-
-	if (actmode == ATR_ACTION_ALTER || actmode == ATR_ACTION_RECOV) {
-		if (!strcasecmp(str, "off") ||
-		    !strcasecmp(str, "low")  ||
-		    !strcasecmp(str, "medium") || !strcasecmp(str, "med") ||
-		    !strcasecmp(str, "high"))
-			return PBSE_NONE;
-		else
+	if (pobj == dflt_scheduler)
```

Comment:
This action should be same irrespective of which scheduler we are setting it on.